components/movementsensor/adxl345/adxl345.go

```diff
@@ -232,6 +358,44 @@ func (adxl *adxl345) writeByte(ctx context.Context, register, value byte) error
 	return handle.WriteByteData(ctx, register, value)
 }
 
+func (adxl *adxl345) configureInterruptRegisters(ctx context.Context) error {
+	if adxl.interruptsEnabled == 0 {
+		return nil
+	}
+	adxl.configuredRegisterValues[IntEnableAddr] = adxl.interruptsEnabled
+	adxl.configuredRegisterValues[IntMapAddr] = adxl.interruptsMap
+	for key, value := range defaultRegisterValues {
```

Comment:
This will iterate through in random order. Suppose we set the `interruptsEnabled` register before the `timeFF` register or something. Will that cause trouble?

Reply:
No, that shouldn't be a problem